examples/web/postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    require('postcss-prefixer')({
+      prefix: 'jobs-',
```

Comment:
its a great idea. but will it be safe to use a version without a prefix? Or does each team have to add their own prefix? Or will there be a general prefix for LMC?

Reply:
That depends on your project. If you use any other CSS frameworks along with Spirit (like Bootstrap), you'll be better off with a prefix. However it's safe for projects based purely on Spirit.